modules/cpmso-yaml-failure-domain-aws.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-aws_{context}"]
+= Sample AWS failure domain configuration
+
+The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
+
+.Sample AWS failure domain values
+[source,yaml]
+----
+failureDomains:
+  aws:
+  - placement:
+      availabilityZone: <aws_region_1> <a>
+    subnet:
+      filters:
+      - name: tag:Name <b>
+        values:
+        - <cluster_id>-private-<aws_region_1> <c>
+      type: Filters
+  - placement:
+      availabilityZone: <aws_region_2> <d>
+    subnet:
+      filters:
+      - name: tag:Name <b>
+        values:
+        - <cluster_id>-private-<aws_region_2> <e>
+      type: Filters
+  platform: AWS <f>
+----
+<a> Specifies an AWS region for the first failure domain.
+
+<b> #Does this need any information?#
+
+<c> Specifies the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster and the AWS region for the first failure domain.
+
+<d> Specifies an AWS region for an additional failure domain.
```

Comment:
Availability zone not region, availability zone is a subset of the region and we don't allow you to vary the region